### DISCUSSION TITLE
阿里更换了token refresh域名？websv.aliyundrive.com貌似已下线了

### DISCUSSION THREAD

**Discussion by galaxy2000** [messense/aliyundrive-webdav#556] [Q&A] [answered]
问下大家，openwrt x86_64下token refresh还能正常使用吗？

前两天突然发现webdav服务无法正常使用，通过日志排查发现websv.aliyundrive.com域名dns无法正常解析了，从而导致token无法正常刷新。

已排除是网络环境的问题，插件版本升级到最新1.8.9也一样报错。

手机和电脑多个不同环境、网络下都测试过，curl https://websv.aliyundrive.com/token/refresh 均无法正常打开。

通过dig也一样是无法正常解析。

**Comment by galaxy2000** [comment, marked as answer]
神奇了，刚升级到1.9.0之后居然好了  -_-|||  

除了opkg remove插件 和 /etc/config/aliyundrive-webdav、aliyundrive-webdav-opkg两个配置文件，再重新安装，啥也没操作呀....

**Reply by messense** [reply]
村通网？https://github.com/messense/aliyundrive-webdav/issues/470

**Reply by messense** [reply]
说明你之前安装的版本实际上低于 1.5.1.

**Reply by galaxy2000** [reply]
低于1.5.1版本不至于，我通过opkg list-installed 反复check过了的，确定是升级到了1.8.9版本。

我在怀疑是不是openwrt卸载的时候没把相关的配置文件也一并删除掉？导致新版插件使用了旧版(1.5.1以下)的配置问题？

**Reply by messense** [reply]
跟配置没关系，websv.aliyundrive.com 这个域名是写死在 binary 中的。

**Comment by puppet-team** [comment]
换    https://api.aliyundrive.com/token/refresh
